src/redux/reducers/filters.ts

```diff
@@ -15,23 +29,42 @@ export const initialState: FiltersState = {
         servings: 0
     }
 }
+>>>>>>> 210f6c3dfad3a7a3bd5b2ae26993d60b795ed086
```

Comment:
It looks like you missed a merge conflict. You would want to accept your current changes where userId is removed